tests/rptest/tests/cloud_storage_usage_test.py

```diff
@@ -140,6 +180,7 @@ def test_cloud_storage_usage_reporting(self):
         producers_done = lambda: all([p.is_complete() for p in producers])
         while not producers_done():
             self._check_usage(timeout_sec=5)
+            self._check_usage_mgr(timeout_sec=30)
```

Comment:
This really big timeout could greatly reduce the number of  `_check_usage` checks that this test performs. Would it be possible to tighten it?

Also, what's the value in checking the cloud storage usage that's reported via the health monitor in this test?
It feels like this would be testing the health monitor rather than the cloud storage usage stuff.

Reply:
The `kafka::usage_manager` takes the values from the health monitor and places them in buckets. The smallest window size is 1 second and then there is another refresh rate for health monitor, i could try to reduce these more, but going too small was causing issues with startup (when health_monitor refresh rate is around 1 second) or test failures because enough time hasn't passed for the latest value to make its way to the `usage_monitor`

Reply:
Moved into separate test